### PR TITLE
Adds B17

### PR DESCRIPTION
**Pull request by Kuro-ai**
## About The Pull Request

Adds the B17 grenadier heavy armor,makes grenadier not share a kit with b18 allowing both minigunner and grenadier to be picked in the same match

## Why It's Good For The Game

Grenadier plays nothing like minigunner,they shouldn't be sharing a kit or sharing the same armor.this adds the B17 which is explosion immune but has no autodoc and separate both kits

## Changelog
:cl:
add: Adds b17 to grenadier kit
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -6. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by Pariah919** [comment]
finally

File: code/game/objects/items/storage/marine_boxes.dm [hunk 1]
```diff
@@ -257,8 +257,8 @@
 /obj/item/storage/box/grenade_system/Initialize(mapload, ...)
 	. = ..()
 	new /obj/item/weapon/gun/launcher/m92(src)
-	new /obj/item/storage/belt/grenade/b18(src)
-
+	new /obj/item/storage/belt/grenade/b17(src)
+
```
Comment (Rohesie) [inline code/game/objects/items/storage/marine_boxes.dm]:
```suggestion

```

File: code/modules/clothing/head/helmet.dm [hunk 2]
```diff
@@ -305,6 +305,16 @@
 	anti_hug = 6
 	flags_item_map_variant = (ITEM_JUNGLE_VARIANT|ITEM_ICE_VARIANT|ITEM_ICE_PROTECTION)
 
+/obj/item/clothing/head/helmet/marine/grenadier
+	name = "\improper B17 helmet"
+	desc = "A heavy duty helmet created to compliment the B17 marine armor. Practically explosive proof. Unless you stand next to a nuke or something like that."
```
Comment (Rohesie) [inline code/modules/clothing/head/helmet.dm]:
```suggestion
	desc = "A heavy duty helmet created to complement the B17 marine armor. Practically explosive proof. Unless you stand next to a nuke or something like that."
```

**Comment by TheBimmer231** [comment]
Kuro's great, approved 👍

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-9). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**